spec/abilities/generic_file_abilities_spec.rb

```diff
@@ -16,46 +16,43 @@
       describe 'creator of object' do
         let(:creating_user) { user }
         let(:current_user) { user }
-        it do
-          should be_able_to(:create, GenericFile.new)
-          should be_able_to(:read, generic_file)
-          should be_able_to(:update, generic_file)
-          should_not be_able_to(:delete, generic_file)
-        end
+        it { is_expected.to be_able_to(:create, GenericFile.new) }
+        it { is_expected.to be_able_to(:versions, generic_file) }
+        it { is_expected.to be_able_to(:read, generic_file) }
+        it { is_expected.to be_able_to(:update, generic_file) }
+        it { is_expected.to_not be_able_to(:delete, generic_file) }
       end
 
       describe 'as a repository manager' do
         let(:manager_user) { FactoryGirl.create(:admin) }
         let(:creating_user) { user }
         let(:current_user) { manager_user }
-        it do
-          should be_able_to(:create, ::GenericFile.new)
-          should be_able_to(:read, generic_file)
-          should be_able_to(:update, generic_file)
-          should be_able_to(:destroy, generic_file)
-        end
+        it { is_expected.to be_able_to(:create, GenericFile.new) }
+        it { is_expected.to be_able_to(:versions, generic_file) }
+        it { is_expected.to be_able_to(:read, generic_file) }
+        it { is_expected.to be_able_to(:update, generic_file) }
+        it { is_expected.to be_able_to(:destroy, generic_file) }
       end
 
       describe 'another authenticated user' do
         let(:creating_user) { FactoryGirl.create(:user) }
         let(:current_user) { user }
-        it do
-          should be_able_to(:create, ::GenericFile.new)
-          should_not be_able_to(:read, generic_file)
-          should_not be_able_to(:update, generic_file)
-          should_not be_able_to(:delete, generic_file)
-        end
+
+        it { is_expected.to be_able_to(:create, GenericFile.new) }
```

Comment:
You've transformed 1 spec into 5 here. This is going to cause the test suite to run slower. Are you sure this is necessary?